ydb/core/tx/datashard/datashard_ut_write.cpp

```diff
@@ -194,6 +194,50 @@ Y_UNIT_TEST_SUITE(DataShardWrite) {
         }
     }
 
+    Y_UNIT_TEST(ImmediateEmptyTxId) {
+        auto [runtime, server, sender] = TestCreateServer();
+
+        TShardedTableOptions opts;
+        auto [shards, tableId] = CreateShardedTable(server, sender, "/Root", "table-1", opts);
+        const ui64 shard = shards[0];
+        const ui32 rowCount = 3;
+
+        const ui64 startTxId = 2;
+
+        Cout << "========= Send several immediate write with txId = 0 =========\n";
+        for(ui32 i = 0; i < 2; ++i)
+        {
+            const auto writeResult = Upsert(runtime, sender, shard, tableId, opts.Columns_, rowCount, 0, NKikimrDataEvents::TEvWrite::MODE_IMMEDIATE);
```

Comment:
I think EvWrite must have txid !=0 or locktxid != 0. If both are zero then datashard should return error.

Reply:
Remove this test